service_container.rst

```diff
@@ -849,7 +861,7 @@ But, if you *do* need to make a service public, override the ``public`` setting:
             # ... same code as before
 
             # explicitly configure the service
-            App\Service\MessageGenerator:
+            App\SomePublicService:
```

Comment:
I don't like this name, any idea is welcome.

Reply:
@HeahDude naming is hard :)

Just a thought, what about `App\Services\MyPublicService` or similar variations?

Reply:
Changed for `Acme\PublicService`